Prioritize Runtime in Software docs (#954)
astronomer/docs: software/customize-image.md

1. In your Astro project, create a duplicate of your `Dockerfile` and name it `Dockerfile.build`.

2. In `Dockerfile.build`, add `AS stage` to the `FROM` line which specifies your Astronomer image. For example, if you use Astro Runtime 5.0.6, your `FROM` line would be:

   ```text
   FROM quay.io/astronomer/runtime:5.0.6-base AS stage1
   ```

  :::caution

>   Make sure to use the `-base` version the Astro Runtime image that you want to customize. This image tag is built to be customizable and does not include default build logic. For more information, see [Distributions](runtime-image-architecture.md#distribution).

Reviewer: ```suggestion
  Make sure to use the `-base` version of the Astro Runtime image that you want to customize. This image tag is  customizable and does not include default build logic. For more information, see [Distributions](runtime-image-architecture.md#distribution).
```

1. In your Astro project, create a duplicate of your `Dockerfile` named `Dockerfile.build`.

2. In `Dockerfile.build`, add `AS stage` to the `FROM` line which specifies your Runtime image. For example, if you use Runtime 5.0.6, your `FROM` line would be:

   ```text
   quay.io/astronomer/astro-runtime:5.0.6-base AS stage1
   ```

   :::caution

>    Make sure to use the `-base` version the Astro Runtime image that you want to customize. This image tag is built to be customizable and does not include default build logic. For more information, see [Distributions](runtime-image-architecture.md#distribution).

Reviewer: ```suggestion
    Make sure to use the `-base` version of the Astro Runtime image that you want to customize. This image tag is  customizable and does not include default build logic. For more information, see [Distributions](runtime-image-architecture.md#distribution).
```
